cms/templates/textbooks.html

```diff
@@ -36,11 +38,39 @@
 
 <%block name="content">
   <div class="wrapper-mast wrapper">
+    % if context_course:
+      <%
+        pages_and_resources_mfe_enabled = ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND.is_enabled(context_course.id)
```

Comment:
I'd use `foo_bar_url =get_pages_and_resources_url` method here which actually checks for `ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND` and returns the url. I would avoid an additional db call here.